networkx/algorithms/centrality/tests/test_betweenness_centrality_subset.py

```diff
@@ -116,6 +116,45 @@ def test_diamond_multi_path(self):
         for n in sorted(G):
             assert b[n] == pytest.approx(expected_b[n], abs=1e-7)
 
+    def test_normalized_p2(self):
+        G = nx.Graph()
+        nx.add_path(G, range(2))
+        b_answer = {0: 0, 1: 0.0}
+        b = nx.betweenness_centrality_subset(
+            G, sources=[0], targets=[1], normalized=True, weight=None
+        )
+        for n in sorted(G):
+            assert b[n] == pytest.approx(b_answer[n], abs=1e-7)
+
+    def test_normalized_P5_directed(self):
+        G = nx.DiGraph()
+        nx.add_path(G, range(5))
+        b_answer = {0: 0, 1: 1.0 / 12.0, 2: 1.0 / 12.0, 3: 0, 4: 0, 5: 0}
+        b = nx.betweenness_centrality_subset(
+            G, sources=[0], targets=[3], normalized=True, weight=None
+        )
+        for n in sorted(G):
+            assert b[n] == pytest.approx(b_answer[n], abs=1e-7)
+
+    def test_weighted_graph(self):
```

Comment:
Where was the weighted test case derived? Did you compute this by hand or was this adapted from one of the other cases?

Reply:
Hi @rossbar thank you so much for the review:) As a first contributor here, I read many of the relevant codes of betweenness and betweenness subset in networkx. The weighted test case I used here is as same as the one in both test_load_centrality.py and test_betweenness_centrality.py and I calculated the betweenness centrality by myself. Previously I thought it was the sample of weighted graph. I will initialize a new weighted graph if it cannot be used here.